test/with-default-focused-input/AutosuggestApp.js

```diff
@@ -0,0 +1,71 @@
+import React, { Component } from 'react';
+import sinon from 'sinon';
+import Autosuggest from '../../src/Autosuggest';
+import languages from '../plain-list/languages';
+import { escapeRegexCharacters } from '../../demo/src/components/utils/utils.js';
+
+const getMatchingLanguages = value => {
```

Comment:
I see that this helper is copy/pasted to all tests. If you want to give more love to this repo, one idea could be to factor this out to a perhaps `../plain-list/languages`.

Reply:
Yes, that makes sense. I can try to do it on another PR, tidying up a bit the tests.